app/components/support_interface/application_choice_component.rb

```diff
@@ -174,6 +174,17 @@ def visible_over_register_api?
       ).find_by(id: application_choice.id).present?
     end
 
+    def change_course_choice_link
+      return {} unless FeatureFlag.inactive?(:change_course_details_before_offer) && @application_choice.application_form.editable? && ApplicationStateChange::DECISION_PENDING_STATUSES.include?(application_choice.status.to_sym)
```

Comment:
When I reverted I thought that this line was confusing. But the spec says that we should not show the link if the feature is `active` which is odd because I thought it was the other way around. Does anybody know more about this specific rule?

Reply:
The `:change_course_details_before_offer` relates to the functionality I built for providers which initially removed the need for this support feature to exist so..

if the feature flag is active then the functionality is visible to providers so we can hide the support feature.

The feature flag was removed as part of the feature release so this condition is no longer required

Reply:
Sweet. That's done! ♻️